### PR TITLE
Initial Travis config

### PR DESCRIPTION
**Pull request by tstrass**
## Summary

This replaces the existing travis config with a new one that sets up the environment necessary to run tests on measures using our tools, fhir-bundle-calculator, and fhir-patient generator. I decided to replace what was there rather than just add to it, because A) their Travis CI doesn't seem to even be running in their repo, B) it was broken when I tried to run it in this repo, and C) there were challenges to getting both ruby and npm installed properly in the same travis config.

For now all this does is prepare the environment for executing tests on measures. In order to get there, I put in a `make` call to fhir-patient-generator and addressed issues by adjusting the travis config until that call succeeded. I have since removed the `make` call.

## To Test

- Check out the Travis job that runs in this PR and verify that it's working as expected.
- In `test_measures.sh`, you could add a `make MEASURE_DIR=<dir>` call, push that change to a separate branch, and then check out the travis job that runs to confirm it works.

### CODE REVIEW

**Comment by okeefm** [comment]
Looking at the code and the travis logs, this looks good. It might at some point be worth suppressing some of the seed-cqf-ruler output, as it makes the travis output very long and therefore hard to read. Up to you whether you want to do that in this PR or make a separate task for it.

File: travis/test_measures.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd fhir-patient-generator
+
+make MEASURE_DIR=EXM_130
```
Comment (okeefm) [inline travis/test_measures.sh]: Should this line be here, since you said in the PR description "I have since removed the make call" ?

Comment (tstrass) [reply travis/test_measures.sh]: My bad failed to commit that. Updated with a force push

**Review by tstrass — COMMENTED** [review]
> Looking at the code and the travis logs, this looks good. It might at some point be worth suppressing some of the seed-cqf-ruler output, as it makes the travis output very long and therefore hard to read. Up to you whether you want to do that in this PR or make a separate task for it.

I'd def say separate task. We can discuss it at backlog groom.

**Comment by tstrass** [comment]
Given recent discussion regarding TACOSTRAT-399, it seems this will be abandoned for the time being, but we will want essentially the same thing in the new travis config in the connectathon repo. Since these two tasks are in review simultaneously, I say let's just go ahead and review this here, and then one of the new tasks to come out of 399 can be to get the travis config set up in the connectathon fork.

**Comment by mgramigna** [comment]
> Given recent discussion regarding TACOSTRAT-399, it seems this will be abandoned for the time being, but we will want essentially the same thing in the new travis config in the connectathon repo. Since these two tasks are in review simultaneously, I say let's just go ahead and review this here, and then one of the new tasks to come out of 399 can be to get the travis config set up in the connectathon fork.

Agreed. This is easily portable and as it stands right now, will likely be almost the exact same thing in `connectathon`

**Review by mgramigna — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
lgtm

**Pull request merged**